### PR TITLE
without this the installation won't function as described (missing m4 file etc)

### PR DESCRIPTION
**Pull request by bug0001**
as requested, without this the installation won't function as described (missing m4 file etc)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -21,10 +21,11 @@ instructions <DEVELOPMENT.rst>`_.
 Get OpenSlides
 --------------
 
-First, you have to clone this repository::
+First, you have to clone this repository, then move to the latest stable branch::
```
Comment (emanuelschuetze) [inline README.rst:24]: better: "... then _checkout_ to the latest stable branch:"

**Comment by emanuelschuetze** [comment]
Please use a better and shorter commit message, e.g. "Updated install instructions in Readme"

File: README.rst [hunk 2]
```diff
@@ -21,10 +21,11 @@ instructions <DEVELOPMENT.rst>`_.
 Get OpenSlides
 --------------
 
-First, you have to clone this repository::
+First, you have to clone this repository, then checkout to the latest stable branch::
 
     git clone https://github.com/OpenSlides/OpenSlides.git --recurse-submodules
     cd OpenSlides/
+    git checkout remotes/origin/stable/3.4.x
```
Comment (jsangmeister) [inline README.rst:28]: Nitpick, but I think
```suggestion
    git checkout stable/3.4.x
```
should be enough, shouldn't it?

Comment (bug0001) [reply README.rst:28]: That's one thing, but the bigger issue at the moment is that some tests fail (python installation, dependencies) even with minor modifications.

**Comment by jsangmeister** [comment]
Closing in favor of https://github.com/OpenSlides/OpenSlides/pull/6359

**Pull request closed without merging**